### ISSUE TITLE
Importer is Broken after latest update

### ISSUE THREAD

**Issue by sveken** [Yellow-Dog-Man/Resonite-Issues#1132]
### Describe the bug?

When importing models such as fbx or GLB, there is no import window and the file is just dumped into the world. 
Images appear to be fine.

### To Reproduce

Copy a FBX or GLB into the world, the file will just be dumped in, with no importer window
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/19641755/71e45c6c-71d5-46f7-a874-9e5c0539d043)

### Expected behavior

It pops up and asks if this is a model.

### Screenshots

![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/19641755/c45f56e9-eaec-4368-a0cc-aeb983871677)

### Resonite Version Number

2025.1.12.1336

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Desktop

### Log Files

[THICC5000 - 2024.1.12.1336 - 2024-01-13 10_33_24.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13924995/THICC5000.-.2024.1.12.1336.-.2024-01-13.10_33_24.log)

### Reporters

Gabriel192
rawpie

**Comment by BigRedWolfy** [comment]
https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/1133

Appending mine here

**Comment by sveken** [comment]
Using Import file on an already dropped in model also does not work.
https://streamable.com/maz8vd

**Comment by Charizmare** [comment]
Can confirm that I'm facing the same issues. Tried an .fbx, .obj and .blend all with no import dialog and they also don't import when tried via the raw file. Below I've attached a video of the results and my log as well. 

https://youtu.be/BuePlMs_wLE
[CHORMATIC-ABERR - 2024.1.12.1336 - 2024-01-12 18_42_27.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13925076/CHORMATIC-ABERR.-.2024.1.12.1336.-.2024-01-12.18_42_27.log)
